modules/webbrowser/webbrowsermodule.cpp

```diff
@@ -162,4 +168,70 @@ WebBrowserModule::~WebBrowserModule() {
     app_->waitForPool();
     CefShutdown();
 }
+    
+std::string WebBrowserModule::getDataURI(const std::string& data, const std::string& mime_type) {
+    return "data:" + mime_type + ";base64," +
+    CefURIEncode(CefBase64Encode(data.data(), data.size()), false)
+    .ToString();
+}
+    
+std::string WebBrowserModule::getCefErrorString(cef_errorcode_t code) {
+#define CASE(code) \
```

Comment:
why use a define for this small snippet?

Reply:
Better suggestion?

Reply:
just write out the code is's only a few chars more...